tests/unit/test_kinesis.py

```diff
@@ -11,11 +11,15 @@
 class KinesisListenerTest(unittest.TestCase):
 
     def test_describe_stream_summary_is_redirected(self):
-        describe_stream_summary_header = {'X-Amz-Target': 'Kinesis_20131202.DescribeStreamSummary'}
+        if config.KINESIS_PROVIDER == 'kinesalite':
```

Comment:
I only want the proxy request for this to run for kinesalite. So this just returns true if the KINESIS_PROVIDER is kinesis-mock